keps/sig-network/3698-multi-network/README.md

```diff
@@ -0,0 +1,363 @@
+As a Virtual Machine -based compute platform provider that I run on top of
+Kubernetes and Kubevirt I need the ability to add/remove Network Interfaces to
+existing VMs without re-creating the VM. This would not be applicable for the
+Primary PodNetwork.
+
+<p align="center">
+  <img src="mn-story-8.png?raw=true" alt="multi-network story 8 network interface hot-plug"/>
+</p>
+
+#### Story #9
+As an infrastructure administrator I wish to support IPv6 SLAAC in the
+Kubernetes Pods. SLAAC's nature is to dynamically assign IP addresses and change
+them over time. Those should be reflected in Pod status. This would not be
+applicable for the Primary PodNetwork.
+
+### Requirements
+
+Below requirements are divided into phases. Each phase will be in its own design
+section of this KEP that will be defined and implemented incrementally.
+
+#### Phase I (base API and reference in Pod)
+1. Multi-network should not change the behavior of existing clusters where no 
+multi-network constructs are configured/used.
+2. Multi-network needs to be represented explicitly as an API construct in
+Kubernetes as it fundamentally ties in with other APIs in the Kubernetes
+networking model, extending their functionality.
+   * We will introduce an PodNetwork API object that represents the existing
+     infrastructure’s networking.
+3. PodNetwork functions as a handle that decouples the implementation
+(i.e. infrastructure/network admins) from the consumer (cluster user). This
+abstraction follows the Kubernetes model of decoupling intent from implementation.
+4. PodNetwork shall not define any implementation specific parameters in its
+specification.
+5. PodNetwork shall provide cluster generic options only.
+6. PodNetwork can reference to implementation-specific parameters.
+7. PodNetwork is the object Application developer will reference in their
+workflows (e.g. Pod, Service).
+8. Cluster Default PodNetwork is the PodNetwork the cluster has been created
+with initially.
+9. Cluster Default PodNetwork cannot be removed, but can be replaced.
+10. Pods can reference one or more PodNetworks when trying to attach to them.
+    * This list is explicit and all the PodNetworks that has to be available in
+      the Pod has to be listed.
+11. Workloads access to PodNetwork is controlled via RBAC configuration.
+12. When Pods does not specify any reference to PodNetwork it connects to Cluster
+Default PodNetwork (network the cluster has been created with).
+13. Pod shall be able to provide additional configuration on how it attaches to
+a PodNetwork.
+14. Every Pod connected to specific PodNetwork must have connectivity within
+that network across the Cluster.
+15. A Pod connected to specific PodNetwork may or may not have cross 
+connectivity between different PodNetworks.
+16. Pods attached to PodNetworks are connected to each other in a manner 
+defined by the PodNetwork implementation.
+17. Implementation-agnostic PodNetwork Interface information for each attachment
```

Comment:
what type of information? a few examples?

Reply:
IP, PodNetwork name, Interface name etc. updated the text.